src/reference/01-General-Info/05-Using-Sonatype.md

```diff
@@ -140,61 +198,57 @@ This might be advantageous if those keys are used also by other plugins
 duplicate entries in the final POM file, leading to a rejection in
 Sonatype's staging process.
 
-*The full format of a pom.xml file is
-[outlined here](https://maven.apache.org/pom.html).*
-
-### Fourth - Adding credentials
-
-The credentials for your Sonatype OSSRH account need to be added
-somewhere. Common convention is a `$global_base$/sonatype.sbt` file
-with the following:
 
+To ensure the POMs are generated and pushed:
 ```scala
-credentials += Credentials("Sonatype Nexus Repository Manager",
-                           "oss.sonatype.org",
-                           "<your username>",
-                           "<your password>")
+publishMavenStyle := true
 ```
 
-> *Note: The first two strings must be
-`"Sonatype Nexus Repository Manager"` and `"oss.sonatype.org"` for Ivy
-to use the credentials.*
+Setting repositories to publish to:
+```scala
+publishTo := {
+  val nexus = "https://oss.sonatype.org/"
+  if (isSnapshot.value)
+    Some("snapshots" at nexus + "content/repositories/snapshots")
+  else
+    Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+}
+```
 
-### Finally - Publish
+Not publishing the test artifacts (this is the default):
+```scala
+publishArtifact in Test := false
+```
 
-In sbt, run `publishSigned` and you should see something like the
-following:
+### Third - Publish to the staging repository
 
+To simplify the usage of the Sonatype's Nexus, add the following line to 
+`build.sbt` to import the [sbt-sonatype plugin][sbt-sonatype] to your project:
```

Comment:
Could you mention that sbt-sonatype is a third-party plugin, and that it's not covered by Lightbend subscription.